tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -797,6 +797,86 @@ public function provide_test_subscription_check_detachment() {
+		$notices = new WC_Stripe_Admin_Notices();
+		$notices->subscription_check_detachment();
+
+		remove_filter( 'pre_http_request', $test_request, 10, 3 );
+
+		$this->assertArrayNotHasKey( 'subscription_detached', $notices->notices );
+
+		unset( $_REQUEST );
+		$theorder = $original_order;
+		WC_Stripe_Database_Cache::delete( 'payment_method_for_source_src_123' );
+		delete_option( 'wc_stripe_show_subscription_detached_notice' );
+	}
```

PHPMD (2.15.0)

[error] test_subscription_detached_notice_hidden_for_dismissed_subscription accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)